Gems/Multiplayer/Code/Source/Pipeline/NetworkPrefabProcessor.cpp

```diff
@@ -63,25 +63,8 @@ namespace Multiplayer
                 ->Field("SerializationFormat", &NetworkPrefabProcessor::m_serializationFormat)
             ;
         }
-    }
-
-    static void LoadInstanceFromPrefab(AzToolsFramework::Prefab::Instance& sourceInstance, const PrefabDom& prefab)
-    {
-        using namespace AzToolsFramework::Prefab;
-
-        // convert Prefab DOM into Prefab Instance.
-        if (!PrefabDomUtils::LoadInstanceFromPrefabDom(sourceInstance, prefab))
-        {
-            PrefabDomValueConstReference sourceReference = PrefabDomUtils::FindPrefabDomValue(prefab, PrefabDomUtils::SourceName);
-
-            AZStd::string errorMessage("NetworkPrefabProcessor: Failed to Load Prefab Instance from given Prefab Dom.");
-            if (sourceReference.has_value() && sourceReference->get().IsString() && sourceReference->get().GetStringLength() != 0)
-            {
-                AZStd::string_view source(sourceReference->get().GetString(), sourceReference->get().GetStringLength());
-                errorMessage += AZStd::string::format("Prefab Source: %.*s", AZ_STRING_ARG(source));
-            }
-            AZ_Error("NetworkPrefabProcessor", false, errorMessage.c_str());
-        }
+    static AZStd::unique_ptr<AzToolsFramework::Prefab::Instance> LoadInstanceFromPrefab(const PrefabDom& prefab)
+        if (!PrefabDomUtils::LoadInstanceFromPrefabDom(*sourceInstance, prefab, PrefabDomUtils::LoadFlags::AssignRandomEntityId))
     }
```

Comment:
This doesn't look like valid syntax. Missed save maybe?

Reply:
Sorry that was a mega merge from dev after 2 months. 
Converted to Draft until I fix everything